### PR TITLE
Update gtfspathways.test.ts

### PR DESCRIPTION
**Pull request by susrisha**
GTFSPathways tests are completed
- Reference task [Task 323](https://dev.azure.com/TDEI-UW/TDEI/_workitems/edit/323): Pathways API Tests

### CODE REVIEW

File: src/__tests__/gtfspathways.test.ts [hunk 1]
```diff
@@ -18,65 +35,228 @@ describe("GTFS PATHWAYS API", () => {
...
-    let gtfsPathwaysAPI = new GTFSPathwaysApi(configuration);
+      it('When passed with valid token and page size 5, should return list of files less than or equal to 5', async () => {
 
-    const versions = await gtfsPathwaysAPI.listPathwaysVersions();
+        let gtfsPathwaysAPI = new GTFSPathwaysApi(configuration);
+        let page_size = 5
 
-    expect(Array.isArray(versions.data.versions)).toBe(true);
-  }, 10000);
+        const pathwayFiles = await gtfsPathwaysAPI.listPathwaysFiles(undefined,undefined,undefined,undefined,undefined,undefined,undefined,page_size);
```
Comment (ayydevara) [inline src/__tests__/gtfspathways.test.ts:56]: consider using `null` instead of `undefined`

File: src/__tests__/gtfspathways.test.ts [hunk 2]
```diff
@@ -18,65 +35,228 @@ describe("GTFS PATHWAYS API", () => {
...
+        });
+
+      })
+
+      it('When passed with valid token and recordId, should return record with same recordId', async () => {
+
+        let recordId = '90d81b53e2e54abebd66986d2fdab169';
+        let gtfsPathwaysAPI = new GTFSPathwaysApi(configuration);
+
+        const pathwayFiles = await gtfsPathwaysAPI.listPathwaysFiles(undefined,undefined,undefined,undefined,undefined,recordId);
```
Comment (ayydevara) [inline src/__tests__/gtfspathways.test.ts:82]: consider using `null` instead of `undefined`

File: src/__tests__/gtfspathways.test.ts [hunk 3]
```diff
@@ -18,65 +35,228 @@ describe("GTFS PATHWAYS API", () => {
...
+
+        await expect(pathwaysResponse).rejects.toMatchObject({response:{status:401}});
+
+      })
+
+      it('When passed with valid token and invalid recordId, should return 200 status with no elements', async () =>{
+        let dummyRecordId = 'dummyRecordId';
+        let pathwaysAPI = new GTFSPathwaysApi(configuration);
+
+        const pathwaysResponse = await pathwaysAPI.listPathwaysFiles(undefined,undefined,undefined,undefined,undefined,dummyRecordId);
```
Comment (ayydevara) [inline src/__tests__/gtfspathways.test.ts:108]: consider using `null` instead of `undefined`

File: src/__tests__/gtfspathways.test.ts [hunk 4]
```diff
@@ -1,11 +1,30 @@
...
+        delete data["file"];
+        delete data["meta"];
+        data.set("file", file, fileName);
+        data.set("meta", JSON.stringify(metaToUpload));
+    }
+    return request;
+};
 
   beforeAll(async () => {
     let generalAPI = new GeneralApi(configuration);
```
Comment (priya-badmanji) [inline src/__tests__/gtfspathways.test.ts:30]: unused variable

File: src/__tests__/gtfspathways.test.ts [hunk 5]
```diff
@@ -18,65 +37,228 @@ describe("GTFS PATHWAYS API", () => {
...
 
-    const requestInterceptor = (request, fileName) => {
-      let data = request.data as FormData;
-      let file = data.get("file") as File;
-      delete data["file"];
-      delete data["meta"];
-      data.set("file", file, fileName);
-      data.set("meta", JSON.stringify(metaToUpload));
+        let gtfsPathwaysAPI = new GTFSPathwaysApi(configuration);
+        let stationId = '5e20eb06-a950-4a5d-a9ca-1e390a801b8a';
```
Comment (priya-badmanji) [inline src/__tests__/gtfspathways.test.ts:68]: can be read from seeder.

Comment (priya-badmanji) [inline src/__tests__/gtfspathways.test.ts:68]: can be read from seeder

**Review by priya-badmanji — DISMISSED** [review]

**Review by priya-badmanji — APPROVED** [review]

**Pull request merged**